packages/@vue/cli-plugin-eslint/generator/index.js

```diff
@@ -130,7 +149,8 @@ module.exports.hooks = (api) => {
   })
 }
 
-const applyTS = module.exports.applyTS = api => {
+// exposed for the typescript plugin
+module.exports.applyTS = api => {
```

Comment:
Where is this being used?

Reply:
Used here: https://github.com/vuejs/vue-cli/blob/1c269d22f5c15e313966db8ce7db23d9b0758d8f/packages/%40vue/cli-plugin-typescript/generator/index.js#L63-L79